docs/how-to/forward-logs-to-loki.md

```diff
@@ -0,0 +1,234 @@
+# How to forward logs to Loki
```

Comment:
Let's remove this how-to from this PR. Even if links are broken for a little bit, I think that's cleaner than having to merge them in a particular order.